vllm/v1/core/scheduler.py

```diff
@@ -545,30 +573,31 @@ def update_from_output(
                 new_running.append(request)
                 continue
 
+            num_computed_tokens_with_output = (
+                self.scheduled_req_ids_to_orig_computed_tokens[req_id].popleft(
+                ))
+            if not self.scheduled_req_ids_to_orig_computed_tokens[req_id]:
+                del self.scheduled_req_ids_to_orig_computed_tokens[req_id]
```

Comment:
```suggestion
            self.scheduled_req_ids_to_orig_computed_tokens.discard(req_id)
```

Reply:
`.discard` is an attribute of `set` but this is now a dict.